engine/runtime/action-processor/src/geometry/convex_hull_constructor.rs

```diff
@@ -0,0 +1,209 @@
+use std::collections::HashMap;
+
+use reearth_flow_geometry::algorithm::convex_hull::ConvexHull;
+use reearth_flow_geometry::types::geometry::Geometry2D;
+use reearth_flow_geometry::types::geometry_collection::GeometryCollection;
+use reearth_flow_runtime::node::REJECTED_PORT;
+use reearth_flow_runtime::{
+    channels::ProcessorChannelForwarder,
+    errors::BoxedError,
+    event::EventHub,
+    executor_operation::{ExecutorContext, NodeContext},
+    node::{Port, Processor, ProcessorFactory, DEFAULT_PORT},
+};
+use reearth_flow_types::{Attribute, AttributeValue, Feature, GeometryValue};
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+use serde_json::Value;
+
+use super::errors::GeometryProcessorError;
+
+#[derive(Debug, Clone, Default)]
+pub struct ConvexHullConstructorFactory;
+
+impl ProcessorFactory for ConvexHullConstructorFactory {
+    fn name(&self) -> &str {
+        "ConvexHullConstructor"
```

Comment:
```suggestion
        "ConvexHullAccumulator"
```
https://docs.safe.com/fme/html/FME-Form-Documentation/FME-Transformers/Transformers/hullaccumulator.htm